src/edgepi/utilities/utilities.py

```diff
@@ -60,8 +60,5 @@ def bitstring_from_list(data: list) -> BitArray:
     Returns:
         BitArray: bitstring of bytes ordered from data[0], data[1], ..., data[n-1]
     """
-    code = BitArray()
-    for value in data:
-        next_byte = pack("uint:8", value)
-        code.append(next_byte)
-    return code
+    # bytes() will raise a ValueError if any ints in data are not in range(0, 256)
```

Comment:
range should be (0, 255)

Reply:
This is referring to a python range, not a math range, so the first number is inclusive & the second is exclusive. (Also the error message that occurs when you do `bytes([256])`)
> ValueError: bytes must be in range(0, 256)

fixed https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/423/commits/2521ddcf31ec3a1d45b9ee69ea81dee3b99fd654